### PR TITLE
rule country sample

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

File: 06-Rules/src/utils/AuthService.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import { isTokenExpired } from './jwtHelper'
+import Auth0Lock from 'auth0-lock'
+
+export default class AuthService extends EventEmitter {
+  constructor(clientId, domain) {
+    super()
+    this.domain = domain
+    // Configure Auth0
+    this.lock = new Auth0Lock(clientId, domain, {
+      additionalSignUpFields: [{
```
Comment (flaviogranero) [inline 06-Rules/src/utils/AuthService.js]: @alemata what if we simplify this removing additionalSignUpFields and address field from profile details?

File: 06-Rules/src/views/Main/Home/Home.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  static propTypes = {
+    auth: T.instanceOf(AuthService)
+  }
+
+  constructor(props, context) {
+    super(props, context)
+    this.state = {
+      profile: props.auth.getProfile(),
+      editing: false
```
Comment (flaviogranero) [inline 06-Rules/src/views/Main/Home/Home.js]: I think `editing` is not being used as well, and also in 03-User-Profile :P

File: 06-Rules/src/components/Profile/ProfileDetails.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        </Col>
+        <Col md={6}>
+          <h3>Profile Details</h3>
+          <p><strong>Name: </strong> {profile.name}</p>
+          <p><strong>Email: </strong> {profile.email}</p>
+          <p><strong>Nickname: </strong> {profile.nickname}</p>
+          <p><strong>Address: </strong> {address}</p>
+          <p><strong>Created At: </strong> {profile.created_at}</p>
+          <p><strong>Updated At: </strong> {profile.updated_at}</p>
+          <p><strong>Country (added by rule): </strong> {profile.country}</p>
```
Comment (flaviogranero) [inline 06-Rules/src/components/Profile/ProfileDetails.js]: @alemata so country is only added in auth0 dashboard?

File: 06-Rules/src/utils/AuthService.js (hunk 1 above)
Comment (alemata) [inline 06-Rules/src/utils/AuthService.js]: I will also remove updateProfile from here

**Comment by alemata** [comment]
@flaviogranero updated! Thanks!

**Comment by flaviogranero** [comment]
👍 @ntotten

**Comment by alemata** [comment]
👍 @chenkie

**Comment by flaviogranero** [comment]
@mpaktiti this is ready

**Pull request merged**